backend/tests/unit/services/prompt/test_prompt_service.py

```
@@ -167,27 +179,79 @@ async def test_create_prompt_with_defaults(self, prompt_service, mock_file_ops_s
+                            tags=[]
+                        )
+                        mock_get.return_value = PromptMeta(
+                            prompt=expected_prompt,
+                            repo_name=repo_name,
+                            file_path=file_path,
+                            recent_commits=[],
+                            pr_info=None
+                        )
+                        mock_workflow.return_value = None
+                        
+                        result, pr_info = await prompt_service.save_prompt(
```

Ruff (0.14.1)

Unpacked variable `pr_info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)